ocs_sample_library_preview/SDS/SdsInterpolationMode.py

```diff
@@ -1,12 +1,13 @@
 from enum import Enum
 
 
-class SdsStreamMode(Enum):
+class SdsInterpolationMode(Enum):
     """
-    Enum 0-3
+    SdsInterpolationMode 0-5
```

Comment:
should we say "fully inclusive" (with two spaces :D ) to be consistent with the other enum docstrings?

Reply:
Made this consistent with the other enums